pom.xml

```diff
@@ -88,17 +84,13 @@
 
     <distributionManagement>
         <snapshotRepository>
-            <id>snapshots</id>
-            <url>${snapshots.repository.url}</url>
+            <id>ossrh</id>
+            <url>/projects/connexta/transformation/pom.xml</url>
```

Comment:
Are we using ION for any of these?

Reply:
not sure what you mean exactly? If you mean in the repository id then no, since we are using the ossrh repo provided by sonatype